CONTRIBUTING.md

```diff
@@ -0,0 +1,135 @@
+
+### Reporting Bugs
+
+This section guides you through submitting a bug report for the AWS Provider for CloudQuery. Following these guidelines helps maintainers and the community understand your report :pencil:, reproduce the behavior :computer: :cloud:, and find related reports :mag_right:.
+
+Before creating bug reports, please check [this list](#before-submitting-a-bug-report) as you might find out that you don't need to create one. When you are creating a bug report, please [include as many details as possible](#how-do-i-submit-a-good-bug-report). Fill out [the required template](.github/ISSUE_TEMPLATE/bug_report.md), the information it asks for helps us resolve issues faster.
+
+> **Note:** If you find a **Closed** issue that seems like it is the same thing that you're experiencing, open a new issue and include a link to the original issue in the body of your new one.
+
+#### Before Submitting A Bug Report
+* **Determine [which repository the problem should be reported in](#break-down-of-responsibilities-and-repositories)**.
+* **Perform a [cursory search](https://github.com/cloudquery/cq-provider-aws/issues)** to see if the problem has already been reported. If it has **and the issue is still open**, add a comment to the existing issue instead of opening a new one.
+
+#### How Do I Submit A (Good) Bug Report?
+
+Bugs are tracked as [GitHub issues](https://guides.github.com/features/issues/). After you've determined [which repository](#break-down-of-responsibilities-and-repositories) your bug is related to, create an issue on that repository and provide the following information by filling in [the template](.github/ISSUE_TEMPLATE/bug_report.md).
+
+Explain the problem and include additional details to help maintainers reproduce the problem:
+
+* **Use a clear and descriptive title** for the issue to identify the problem.
+* **Describe the Bug** in as many details as possible. For example, start by explaining how and where you are running Cloudquery (local machine, cloud service, docker, k8s, CI Pipeline, etc)
+* **Provide specific examples to demonstrate the steps**. Include links to gists and or files, or copy/pasteable snippets to help give context to the issue. If you're providing snippets in the issue, use [Markdown code blocks](https://help.github.com/articles/markdown-basics/#multiple-lines).
+* **Explain which behavior you expected to see instead and why.**
+* **Inlcude (sanitized) log output** execute cloudquery with the `--enable-console-log` and `-v` flags to get all of the debug information
+
+Provide more context by answering these questions:
+
+Include details about your configuration and environment:
+
+* **Which version of Cloudquery are you using?** You can get the exact version by running `cloudquery version`
+* **Whats in your config.hcl**? Include as much of the config.hcl as possible. This will allow the community to work to reproduce the issue and identify work arounds and or create fixes to your issues
```

Comment:
```suggestion
* **What's in your config.hcl**? Include as much of the `config.hcl` as possible. This will allow the community to work to reproduce the issue and identify workarounds and/or create fixes to your issues
```